package.json

```diff
@@ -72,6 +73,7 @@
     "@adobe/reactor-object-assign": "^1.0.0",
     "@adobe/reactor-query-string": "^1.0.0",
     "css.escape": "^1.5.1",
+    "inquirer": "^9.2.23",
```

Comment:
Can this be a devDependency?

Reply:
From what I saw on internet, npx doesn't install dev dependencies. I don't know how I can test that in order to be sure.